wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -49,6 +48,21 @@ pub enum WalletCommand {
     /// Close wallet file
     CloseWallet,
 
+    /// Encrypts the private keys with a new password, expects the wallet to be unlocked
+    EncryptPrivateKeys {
+        // Optional new password, if not specified will remove any existing encryption.
+        password: Option<String>,
+    },
+
+    // Unlocks the private keys for usage.
+    UnlockPrivateKeys {
```

Comment:
Why would someone attempt to "unlock" the wallet without providing a password?

Reply:
It was in the case when there is no password, but I guess it doesn't make sense to lock it without a password as nothing is encrypted. 
I will change it to return an error if lock is called when there is no password already set, and remove the optional from the unlock.